@jpmckinney do you know why `test_search` could be failing locally when the PR builds successfully?

```
========================================================================== FAILURES ==========================================================================
______________________________________________________ test_search[en-found \\d+ page\\(s\\) matching] _______________________________________________________

browser = <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>, server = 'http://localhost:8331/', lang = 'en'
regex = 'found \\d+ page\\(s\\) matching'

    @pytest.mark.parametrize('lang,regex', test_search_params)
    def test_search(browser, server, lang, regex):
        browser.get('{}{}'.format(server, lang))
        search_box = browser.find_element_by_id('rtd-search-form').find_element_by_tag_name('input')
        search_box.send_keys('tender\n')
        time.sleep(2)
>       assert re.search(regex, browser.find_element_by_tag_name('body').text)
E       assert None
E        +  where None = <function search at 0x7fa25aecfb80>('found \\d+ page\\(s\\) matching', '            \n\n\n           \n            \n            Language\n            English\n            Español\n        ...ch\nCopyright (c) Open Contracting Partnership, and released under the Apache License 2.0, source available on GitHub.')
E        +    where <function search at 0x7fa25aecfb80> = re.search
E        +    and   '            \n\n\n           \n            \n            Language\n            English\n            Español\n        ...ch\nCopyright (c) Open Contracting Partnership, and released under the Apache License 2.0, source available on GitHub.' = <selenium.webdriver.remote.webelement.WebElement (session="25ffe6f3b2dd015edfbe385464507edd", element="1919841a-53ef-4b4f-84be-0150e37fd572")>.text
E        +      where <selenium.webdriver.remote.webelement.WebElement (session="25ffe6f3b2dd015edfbe385464507edd", element="1919841a-53ef-4b4f-84be-0150e37fd572")> = <bound method WebDriver.find_element_by_tag_name of <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>>('body')
E        +        where <bound method WebDriver.find_element_by_tag_name of <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>> = <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>.find_element_by_tag_name

tests/test_common.py:35: AssertionError
-------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------
127.0.0.1 - - [12/Feb/2021 11:20:33] "GET /en/search/?q=tender&check_keywords=yes&area=default HTTP/1.1" 200 -
127.0.0.1 - - [12/Feb/2021 11:20:33] "GET /en/_static/js/search.js HTTP/1.1" 200 -
__________________________________________________ test_search[es-encontr\xf3 \\d+ p\xe1gina\\(s\\) acorde] __________________________________________________

browser = <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>, server = 'http://localhost:8331/', lang = 'es'
regex = 'encontró \\d+ página\\(s\\) acorde'

    @pytest.mark.parametrize('lang,regex', test_search_params)
    def test_search(browser, server, lang, regex):
        browser.get('{}{}'.format(server, lang))
        search_box = browser.find_element_by_id('rtd-search-form').find_element_by_tag_name('input')
        search_box.send_keys('tender\n')
        time.sleep(2)
>       assert re.search(regex, browser.find_element_by_tag_name('body').text)
E       assert None
E        +  where None = <function search at 0x7fa25aecfb80>('encontró \\d+ página\\(s\\) acorde', '            \n\n\n           \n            \n            Language\n            English\n            Español\n        ...da\nCopyright (c) Open Contracting Partnership, and released under the Apache License 2.0, source available on GitHub.')
E        +    where <function search at 0x7fa25aecfb80> = re.search
E        +    and   '            \n\n\n           \n            \n            Language\n            English\n            Español\n        ...da\nCopyright (c) Open Contracting Partnership, and released under the Apache License 2.0, source available on GitHub.' = <selenium.webdriver.remote.webelement.WebElement (session="25ffe6f3b2dd015edfbe385464507edd", element="7c6c8d8c-f724-40a1-ac11-1b5add2d2b48")>.text
E        +      where <selenium.webdriver.remote.webelement.WebElement (session="25ffe6f3b2dd015edfbe385464507edd", element="7c6c8d8c-f724-40a1-ac11-1b5add2d2b48")> = <bound method WebDriver.find_element_by_tag_name of <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>>('body')
E        +        where <bound method WebDriver.find_element_by_tag_name of <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>> = <selenium.webdriver.chrome.webdriver.WebDriver (session="25ffe6f3b2dd015edfbe385464507edd")>.find_element_by_tag_name

tests/test_common.py:35: AssertionError
-------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------
127.0.0.1 - - [12/Feb/2021 11:20:36] "GET /es/search/?q=tender&check_keywords=yes&area=default HTTP/1.1" 200 -
127.0.0.1 - - [12/Feb/2021 11:20:36] "GET /es/_static/js/search.js HTTP/1.1" 200 -

```

--- a/docs/reference/changelog.md
+++ b/docs/reference/changelog.md
@@ -17,7 +17,6 @@
 
 * [#282](https://github.com/open-contracting/infrastructure/pull/282) - update codes shared with OCDS for PPPs 1.0.0-beta2 and OCDS 1.1.4.
 * [#264](https://github.com/open-contracting/infrastructure/issues/264) - add a field and class for natural persons.
-* [#273](https://github.com/open-contracting/infrastructure/issues/273) - add 'paymentCertificate' code to documentType codelist.
 
 #### documentType codelist
 
@@ -36,6 +35,7 @@ Added:
 * [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'.
 * [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'.
 * [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'.
+* [#273](https://github.com/open-contracting/infrastructure/issues/273) 'paymentCertificate'.
 
 Removed:
 

--- a/docs/reference/changelog.md
+++ b/docs/reference/changelog.md
@@ -4,12 +4,12 @@
 
 ### Documentation
 
-* [#217](https://github.com/open-contracting/infrastructure/issues/217) - remove repeated 'OCDS:' in mapping documentation
-* [#268](https://github.com/open-contracting/infrastructure/issues/268), [#269](https://github.com/open-contracting/infrastructure/issues/269) replace 'finalAudit' with 'technicalAuditReport' and 'financialAuditReport' in mapping
+* [#217](https://github.com/open-contracting/infrastructure/issues/217) - remove repeated 'OCDS:' in mapping documentation.
+* [#268](https://github.com/open-contracting/infrastructure/issues/268), [#269](https://github.com/open-contracting/infrastructure/issues/269) replace 'finalAudit' with 'technicalAuditReport' and 'financialAuditReport' in mapping.
 
 ### Schema
 
-* [#277](https://github.com/open-contracting/infrastructure/issues/277) - add `forecasts` and `metrics` which can be used to publish implementation progress reports
+* [#277](https://github.com/open-contracting/infrastructure/issues/277) - add `forecasts` and `metrics`, which can be used to publish implementation progress reports.
 * [#282](https://github.com/open-contracting/infrastructure/pull/282) - update fields shared with OCDS for PPPs 1.0.0-beta2 and OCDS 1.1.4.
 * [#273](https://github.com/open-contracting/infrastructure/issues/273) - add `contractingProcesses/summary/transactions`, which can be used to publish disbursement records.
 
@@ -23,40 +23,40 @@
 
 Changed:
 
-* [#261](https://github.com/open-contracting/infrastructure/issues/261) Update description of 'feasibilityStudy' code to include "project"
-* [#267](https://github.com/open-contracting/infrastructure/issues/267) Update description of 'completionCertificate' code to include "project"
+* [#261](https://github.com/open-contracting/infrastructure/issues/261) Update description of 'feasibilityStudy' code to include "project".
+* [#267](https://github.com/open-contracting/infrastructure/issues/267) Update description of 'completionCertificate' code to include "project".
 
 Added:
 
-* [#263](https://github.com/open-contracting/infrastructure/issues/263) 'resettlementPlan'
-* [#265](https://github.com/open-contracting/infrastructure/issues/265) 'financialAgreement'
-* [#266](https://github.com/open-contracting/infrastructure/issues/266) 'budgetAmendmentApproval'
-* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'technicalAuditReport'
-* [#269](https://github.com/open-contracting/infrastructure/issues/269) 'financialAuditReport'
-* [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'
-* [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'
-* [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'
+* [#263](https://github.com/open-contracting/infrastructure/issues/263) 'resettlementPlan'.
+* [#265](https://github.com/open-contracting/infrastructure/issues/265) 'financialAgreement'.
+* [#266](https://github.com/open-contracting/infrastructure/issues/266) 'budgetAmendmentApproval'.
+* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'technicalAuditReport'.
+* [#269](https://github.com/open-contracting/infrastructure/issues/269) 'financialAuditReport'.
+* [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'.
+* [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'.
+* [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'.
 
 Removed:
 
-* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'finalAudit'
+* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'finalAudit'.
 
 ## [0.9.2] - 2020-06-29
 
 ### Documentation
 
 * [#96](https://github.com/open-contracting/infrastructure/issues/96) - add guidance on providing project identifiers in OCDS data.
-* [#120](https://github.com/open-contracting/infrastructure/issues/120) - add list of registered project identifier prefixes to documentation
+* [#120](https://github.com/open-contracting/infrastructure/issues/120) - add list of registered project identifier prefixes to documentation.
 * [#124](https://github.com/open-contracting/infrastructure/issues/124) - clarify guidance on project identifier prefixes.
 * [#131](https://github.com/open-contracting/infrastructure/issues/131) - replace 'owner' with 'publicAuthority' in mapping.
-* [#133](https://github.com/open-contracting/infrastructure/issues/133) - improve clarity of 'what is a project' in getting started section
+* [#133](https://github.com/open-contracting/infrastructure/issues/133) - improve clarity of 'what is a project' in getting started section.
 * [#136](https://github.com/open-contracting/infrastructure/issues/136) - add project identifier prefix to example file.
 * [#143](https://github.com/open-contracting/infrastructure/issues/143) - update worked example page to describe project package, use non-normative keywords, and edit for clarity.
 * [#143](https://github.com/open-contracting/infrastructure/issues/143) - add data user guide page.
-* [#145](https://github.com/open-contracting/infrastructure/issues/145) - re-order codelist reference page, refer to OCDS and extension documentation for codelists that are shared
+* [#145](https://github.com/open-contracting/infrastructure/issues/145) - re-order codelist reference page, refer to OCDS and extension documentation for codelists that are shared.
 * [#146](https://github.com/open-contracting/infrastructure/issues/146) - add 'publicAuthority' role to example file.
-* [#218](https://github.com/open-contracting/infrastructure/pull/218) - add link to CoST guidance note on OGP commitments
-* [#211](https://github.com/open-contracting/infrastructure/issues/211) - update description of 'publicAuthority' role
+* [#218](https://github.com/open-contracting/infrastructure/pull/218) - add link to CoST guidance note on OGP commitments.
+* [#211](https://github.com/open-contracting/infrastructure/issues/211) - update description of 'publicAuthority' role.
 
 ### Schema
 
@@ -67,24 +67,24 @@ Removed:
 
 #### OC4IDS project schema
 
-* [#127](https://github.com/open-contracting/infrastructure/issues/127) - remove the requirement that linked OCDS releases must be provided in release packages containing only one release. Remove recommendation that OCDS releases are cached from schema and add guidance on caching releases from unreliable sources to implementation guidance
-* [#132](https://github.com/open-contracting/infrastructure/issues/132) - add a publicAuthority organization reference field  
-* [#139](https://github.com/open-contracting/infrastructure/issues/139) - update properties of fields in common with OCDS to version [1.1.4](https://standard.open-contracting.org/1.1/en/schema/changelog/#id1)
+* [#127](https://github.com/open-contracting/infrastructure/issues/127) - remove the requirement that linked OCDS releases must be provided in release packages containing only one release. Remove recommendation that OCDS releases are cached from schema and add guidance on caching releases from unreliable sources to implementation guidance.
+* [#132](https://github.com/open-contracting/infrastructure/issues/132) - add a publicAuthority organization reference field  .
+* [#139](https://github.com/open-contracting/infrastructure/issues/139) - update properties of fields in common with OCDS to version [1.1.4](https://standard.open-contracting.org/1.1/en/schema/changelog/#id1).
 * [#140](https://github.com/open-contracting/infrastructure/issues/140) - update the description of `project/period` to clarify that this field should be used to provide the planned start and end dates during the preparation phase, for comparison with the actual completion date for the project.
-* [#141](https://github.com/open-contracting/infrastructure/issues/141) - clarify that `contractingProcesses/summary/description` is for the contract's *initial* scope of work
-* [#141](https://github.com/open-contracting/infrastructure/issues/141) - remove incorrect guidance about other fields from `contractingProcesses/summary/modifications`
-* [#153](https://github.com/open-contracting/infrastructure/issues/153) - add project/relatedProjects array
-* [#154](https://github.com/open-contracting/infrastructure/issues/154) - add `.requestDate` field to `project/budget` to record the date of the budget request for the project
-* [#156](https://github.com/open-contracting/infrastructure/issues/156) - fix the description of `completion/endDateDetails` to refer to the end date of the *project*, not that of the *contract*
-* [#157](https://github.com/open-contracting/infrastructure/issues/157) - fix spelling and grammar issues
-* [#158](https://github.com/open-contracting/infrastructure/issues/158) - make `contractingProcesses/releases/tag` an array, not a string (bugfix)
-* [#160](https://github.com/open-contracting/infrastructure/issues/160) - describe the components of `project/id`, and link to guidance
-* [#161](https://github.com/open-contracting/infrastructure/issues/161) - removed `contractingProcesses/summary/ocid` because it duplicates `contractingProcesses/id`
+* [#141](https://github.com/open-contracting/infrastructure/issues/141) - clarify that `contractingProcesses/summary/description` is for the contract's *initial* scope of work.
+* [#141](https://github.com/open-contracting/infrastructure/issues/141) - remove incorrect guidance about other fields from `contractingProcesses/summary/modifications`.
+* [#153](https://github.com/open-contracting/infrastructure/issues/153) - add project/relatedProjects array.
+* [#154](https://github.com/open-contracting/infrastructure/issues/154) - add `.requestDate` field to `project/budget` to record the date of the budget request for the project.
+* [#156](https://github.com/open-contracting/infrastructure/issues/156) - fix the description of `completion/endDateDetails` to refer to the end date of the *project*, not that of the *contract*.
+* [#157](https://github.com/open-contracting/infrastructure/issues/157) - fix spelling and grammar issues.
+* [#158](https://github.com/open-contracting/infrastructure/issues/158) - make `contractingProcesses/releases/tag` an array, not a string (bugfix).
+* [#160](https://github.com/open-contracting/infrastructure/issues/160) - describe the components of `project/id`, and link to guidance.
+* [#161](https://github.com/open-contracting/infrastructure/issues/161) - removed `contractingProcesses/summary/ocid` because it duplicates `contractingProcesses/id`.
 * [#182](https://github.com/open-contracting/infrastructure/issues/182) - update validation properties to enforce unique items in arrays and minimum length on required string fields.
 
 ### Codelists
 
-* [#139](https://github.com/open-contracting/infrastructure/issues/139) - update codelists in common with OCDS to version [1.1.4](https://standard.open-contracting.org/1.1/en/schema/changelog/#id1)
+* [#139](https://github.com/open-contracting/infrastructure/issues/139) - update codelists in common with OCDS to version [1.1.4](https://standard.open-contracting.org/1.1/en/schema/changelog/#id1).
 * [#152](https://github.com/open-contracting/infrastructure/issues/152) - add 'expansion' code to projectType codelist.
 
 ## [0.9.1] - 2019-06-17
@@ -114,20 +114,20 @@ In addition to the specific changes to schema and codelists noted below:
 
 ### Schema updates
 
-* `sector` - use projectSector open codelist
-* `ContractingProcess` - add required `id` field
-* `LinkedRelease` - make `id` required
-* `variations` - rename to `modifications`
-* `Location` - add required `id` field
+* `sector` - use projectSector open codelist.
+* `ContractingProcess` - add required `id` field.
+* `LinkedRelease` - make `id` required.
+* `variations` - rename to `modifications`.
+* `Location` - add required `id` field.
 
 ### New codelists
 
-* `projectSector` codelist - add codelist for project sector
+* `projectSector` codelist - add codelist for project sector.
 
 ### Codelist updates
 
-* projectStatus codelist - replace 'construction' with 'implementation'
-* variationType codelist - rename to modificationType
+* projectStatus codelist - replace 'construction' with 'implementation'.
+* variationType codelist - rename to modificationType.
 * partyRole codelist - add OC4IDS codes mentioned in schema and mapping:
   * funder
   * administrativeEntity

--- a/docs/reference/changelog.md
+++ b/docs/reference/changelog.md
@@ -11,11 +11,13 @@
 
 * [#277](https://github.com/open-contracting/infrastructure/issues/277) - add `forecasts` and `metrics` which can be used to publish implementation progress reports
 * [#282](https://github.com/open-contracting/infrastructure/pull/282) - update fields shared with OCDS for PPPs 1.0.0-beta2 and OCDS 1.1.4.
+* [#273](https://github.com/open-contracting/infrastructure/issues/273) - add `contractingProcesses/summary/transactions`, which can be used to publish disbursement records.
 
 ### Codelists
 
 * [#282](https://github.com/open-contracting/infrastructure/pull/282) - update codes shared with OCDS for PPPs 1.0.0-beta2 and OCDS 1.1.4.
 * [#264](https://github.com/open-contracting/infrastructure/issues/264) - add a field and class for natural persons.
+* [#273](https://github.com/open-contracting/infrastructure/issues/273) - add 'paymentCertificate' code to documentType codelist.
 
 #### documentType codelist
 

